### DISCUSSION TITLE
Help with long term streaming

### DISCUSSION THREAD

**Discussion by monkeymademe** [monkeymademe/CamUI#97] [Q&A]
Hey peeps,

I made a change on https://github.com/monkeymademe/picamera2-WebUI/tree/v1.0.5-Pre branch that might address the issue with the feed cutting out after running for a long time. 

Would it be possible to get your kind selfs to give this branch ago and let me know if the situation improves.

@AleksanderKruecken and @RickRoz both said you had issues and with older pi's? I have my pi5 running but I fear that might be fine for a good while. Maybe you would see an issue sooner if this fix does not work

Thanks for the support

**Comment by RickRoz** [comment]
Hi there, i reinstalled everything from your link. It is still going for me.

The problem for me is that i have added the video link as a generic camera in home assistant, every time when i click on the camera in home assistant the feed in the WebUI stops working.

**Reply by monkeymademe** [reply]
I tired generic camera and it did not go well. I gave MPEG IP camera and I got better results. I used the IP and not the host name and you can see from my screenshot I got both HA and the webui working at the same time.

BUT I am on a pi5 and a good network?

<img width="1409" alt="image" src="https://github.com/user-attachments/assets/fe9b9f76-6dd6-4379-9140-efef5389f03a">

**Reply by RickRoz** [reply]
> I tired generic camera and it did not go well. I gave MPEG IP camera and I got better results. I used the IP and not the host name and you can see from my screenshot I got both HA and the webui working at the same time.
> 
> BUT I am on a pi5 and a good network?
> 
> <img alt="image" width="1409" src="https://private-user-images.githubusercontent.com/1418773/372896358-fe9b9f76-6dd6-4379-9140-efef5389f03a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgxNDg5MDYsIm5iZiI6MTcyODE0ODYwNiwicGF0aCI6Ii8xNDE4NzczLzM3Mjg5NjM1OC1mZTliOWY3Ni02ZGQ2LTQzNzktOTE0MC1lZmVmNTM4OWYwM2EucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDVUMTcxNjQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTI4YjY5NjI2OTM5YWVkNGJjMjg2NDgyMGQ4YTU4NWJiZDZjMTBmZjJhYjM3M2RjOWNjZjgwMDlhNzRkZjQ2ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.9QMrjlZ42ADsD0tQ51iAcoa8SUA2FKC03uyOKUBAbio">

Still got the same problem, my network is good that should not be the issue. I think it is just the RPI 4 then.

**Reply by monkeymademe** [reply]
I can dig out my pi4... one thing worth noting on the pi3 and maybe the pi4 don't max out the resolution I have seen issues where feeds stop working simply cause there is not enough resources going around. I had a cm4 running tonight and it was fine maxed out but then it was the best cm4 model... but like I said I have seen pi3s have issues. Maybe try again but make sure the feed capture and sensor modes are both at the lowest setting before ha'ing it
